frontend/src/pages/Home/components/Testimonials/testimonials.js

```diff
@@ -0,0 +1,49 @@
+const testimonials = [
```

Comment:
can you add a comment on top of file saying - `TODO - Remove this file after api gets integrated` ?

Reply:
> can you add a comment on top of file saying - `TODO - Remove this file after api gets integrated` ?

I have added the comment now.